packages/appium/lib/utils.js

```diff
@@ -1,11 +1,12 @@
 import _ from 'lodash';
 import logger from './logger';
-import {processCapabilities, PROTOCOLS} from '@appium/base-driver';
+import {processCapabilities, PROTOCOLS, STANDARD_CAPS} from '@appium/base-driver';
 import {inspect as dump} from 'util';
 import {node} from '@appium/support';
 import path from 'path';
 
 const W3C_APPIUM_PREFIX = 'appium';
+const STANDARD_CAPS_LOWERCASE = new Set([...STANDARD_CAPS].map((cap) => cap.toLowerCase()));
```

Comment:
perhaps Array.of would be more readable than the spread operator in this case?

Reply:
also we can provide a mapping function as the second arg to `from`

Reply:
you mean `Array.from`, but idk, I don't see either as inherently more "readable".  in other words stuff like this is not a clear win and I'd just as soon ignore it 🤷